ext/lmdb-safe/lmdb-safe.cc

```diff
@@ -16,14 +16,14 @@ static string MDBError(int rc)
   return mdb_strerror(rc);
 }
 
-MDBDbi::MDBDbi(MDB_env* env, MDB_txn* txn, const string_view dbname, int flags)
+MDBDbi::MDBDbi(MDB_env* /* env */, MDB_txn* txn, const string_view dbname, int flags)
```

Comment:
I wonder if we could remove this parameter? It looks that way but I'm not familiar with the LMDB code.

Reply:
#12546 is removing tons of dead code from LMDB. Maybe this PR can leave it alone for now :)

Reply:
@Habbie [This](https://github.com/PowerDNS/pdns/pull/12549/commits/90d690242f073dd549918456319b679a8a09134e) and [this](https://github.com/PowerDNS/pdns/pull/12549/commits/2ac1eb53ca405c0143e25781b8159755a7502c6e) might come in useful.

Reply:
those now 404, corrected links: [a](https://github.com/PowerDNS/pdns/commit/90d690242f073dd549918456319b679a8a09134e) and [b](https://github.com/PowerDNS/pdns/commit/90d690242f073dd549918456319b679a8a09134e)

Reply:
picked up, thanks!